DirectEmployers/MyJobs
# [PD-3659] use waffle flag to control access to new site-traffic reporting

## What's this PR do?
Switch to using waffle flag (instead of switch) to give our staff access to the new site-traffic reporting features before rolling out to members.

## Screenshots (if appropriate)

![image](https://user-images.githubusercontent.com/27963853/30182622-7a557558-93e5-11e7-873c-d6dfdde724fb.png)

## How should this be manually tested?
Go to reports-> site-traffic, you should only see three reporting categories (Browser, job-seeker locations and OS) and they should still work as before. 
Create a waffle flag in Django admin, set values according to screenshot above
Click on the search icon next to `Users`, then locate the local user for testing, the user ID will be added. Save the flag. 
Go to reports -> site-traffic, you should see all new report categories on the left side. 

## Any background context you want to provide?
The `Everyone` option at the top overrides other settings, if set to `no`, even whitelisted users won't have access to the flagged feature, so make sure to set it to "Unkown" and then give access to specific users, or user groups.
